tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -223,6 +223,50 @@ ROOT::Experimental::Detail::RPageSinkDaos::CommitSealedPageImpl(DescriptorId_t c
    return result;
 }
 
+std::vector<ROOT::Experimental::RNTupleLocator>
+ROOT::Experimental::Detail::RPageSinkDaos::CommitSealedPageVImpl(std::span<RPageStorage::RSealedPageGroup> ranges)
+{
+   RDaosContainer::MultiObjectRWOperation_t writeRequests;
+   std::vector<ROOT::Experimental::RNTupleLocator> locators;
+   locators.reserve(ranges.size());
```

Comment:
If space in the vector is to be pre-allocated, the number of expected `RNTupleLocator`s is equal to the number of pages described by `ranges`, i.e. the sum of `std::distance(range.fFirst, range.fLast)` for each element in `ranges`.